cli/azd/.vscode/launch.json

```diff
@@ -20,8 +20,8 @@
             "program": "${workspaceFolder}",
 
             // Uncomment these settings to control how `azd` is launched.
-            // "cwd": "${workspaceFolder}",
-            // "args": ["provision"]
+            "cwd": "/home/heaths/src/azcrypto",
+            "args": ["provision", "--parameters", "vaultName=heathskv2"]
```

Comment:
Please remove these from the commit.  
**Pro tip**: we have a special debug flag you can set that will halt azd and allow you to attach to the running go process in VS code to easily debug any command without having to keep updating launch settings.

Example
```bash
AZD_DEBUG=true azd provision --parameters vaultName=foo
```

Looks like this:
<img width="341" alt="image" src="https://github.com/Azure/azure-dev/assets/6540159/11b89d41-f158-4b5a-8dec-a529b95c44d7">

Reply:
You can also use `git update-index` as described at the top of this file to have git just ignore these changes (I personally use the workflow, that @heaths uses here, where you edit launch.config as it allows me to do everything from with in VSCode natively).

Reply:
Oops. I didn't mean to check that in.

Another way is to initially add _.vscode/launch.json_ in a state you want generically, *then* add it to _.gitignore_. It'll be checked out, but changes to it won't be tracked. Though @weikanglim tells me a contribution he made to the Go extension for VSCode should allow us to pass parameters as well.

Reply:
Yeah. I did mention it to the team. vscode-go changes are in nightly, still waiting for the next release before we remove this altogether.